themes/ctl-histologylab/static/scss/base/base.scss

```diff
@@ -0,0 +1,6 @@
+slidepopup .thumbnail, .thumbnail {
```

Comment:
This should be `.slidepopup`, not `slidepopup`, right? unless i'm missing something.

Reply:
Yep, you're correct. Thanks!